docs/src/gui/qtvcp-widgets.adoc

```diff
@@ -857,7 +793,10 @@ For `GcodeDisplay` widget's _default G-code lexer_:
 * *styleColor6 = AxisValue*: Values following XYZABCUVW
 * *styleColor7 = OtherValue*: Values following EFHIJKDQLRPST$
 
-NOTE: For comments, the "OtherValue" color (Color 5) can be used to highlight "print," "debug," "msg," "logopen," "logappend," "logclose" "log," "pyrun," "pyreload" "abort," "probeopen" "probeclose" inside of a parenthesis comment in a line of G-code. As well as "py," if a line that starts with ";py,". Examples: (print, text), (log, text), (msg, text), or (debug, text). Only the last of the examples will be highlighted if there are more than one on the same line.
+NOTE: For comments, the "OtherValue" color (Color 5) can be used to highlight "print," "debug," "msg," "logopen," "logappend," "logclose" "log," "pyrun," "pyreload" "abort," "probeopen" "probeclose" inside of a parenthesis comment in a line of G-code.
+As well as "py," if a line that starts with ";py,".
+Examples: (print, text), (log, text), (msg, text), or (debug, text).
+Only the last of the examples will be highlighted if there are more than one on the same line.
```

Comment:
This you pulled out of the NOTE?

Reply:
Not intentionally.

Reply:
I haven't checked if it's really the case, it only looked so in the GitHub diff rendering...